README.md

```diff
@@ -11,6 +11,8 @@ Membrane Curvature
 MembraneCurvature is an [MDAnalysis] tool to calculate membrane curvature from 
 Molecular Dynamics simulations. 
 
+> **Interested in becoming a maintainer?** We welcome your passion and expertise to help shape and grow this open-source project! Please contact estefania@ojeda-e.com for more details.
+
```

Comment:
@orbeckst I added this comment to look for maintainers. Should I add an MDAnalysis email better?

Reply:
If I were someone interested in maintaining, _you_ would be the person I'd most like to talk to — so I think having you there is best, provided that you'll look at emails.

If you get in contact with someone and need advice or just talk things through then just get in touch with the MDA team. (One advantage you have over many other people is that you know who to talk to because you worked with the MDAnalysis project!)